commit f624393224ea9156cb880bcdaa2afb2e85a6def8
Author: Eric Anholt <eric@anholt.net>
Date:   Fri Mar 9 14:24:05 2018 -0800

    ARM: BCM270X: Add the 18-bit DPI pinmux to the RPI DTs.
    
    This doesn't do anything by default, but trying to put the node in an
    overlay failed for me.
    
    Signed-off-by: Eric Anholt <eric@anholt.net>

diff --git a/arch/arm/boot/dts/bcm270x.dtsi b/arch/arm/boot/dts/bcm270x.dtsi
index 145e455a2a14..27f19225245e 100644
--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -19,6 +19,13 @@
 
 		gpio@7e200000 { /* gpio */
 			interrupts = <2 17>, <2 18>;
+
+			dpi_18bit_gpio0: dpi_18bit_gpio0 {
+				brcm,pins = <0 1 2 3 4 5 6 7 8 9 10 11
+					     12 13 14 15 16 17 18 19
+					     20 21>;
+				brcm,function = <BCM2835_FSEL_ALT2>;
+			};
 		};
 
 		serial@7e201000 { /* uart0 */
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1307_f624393224ea9156cb880bcdaa2afb2e85a6def8.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1307_f624393224ea9156cb880bcdaa2afb2e85a6def8.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
